sys/default_arm7/source/main.c

```diff
@@ -22,13 +52,17 @@ void power_button_callback(void)
 void vblank_handler(void)
 {
     inputGetAndSend();
+#if defined(USE_DSWIFI)
     Wifi_Update();
+#endif
 }
 
-int main(int argc, char *argv[])
```

Comment:
Also, I would leave this the way it was before, even if it's unused. It may be useful some day, and it's better to leave the arguments there because people tend to copy the example code right away and forget about it.

Reply:
```diff
-int main(void)
+int main(int argc, char *argv[])
 {
+    // Unused variables
+    (void)argc;
+    (void)argv;
+    
     // Initialize sound hardware
```

```diff
-int main(void)
+int main(int argc, char *argv[])
 {
     // Initialize sound hardware
```

Which one of these would be preferred? The first one avoids unused variable warnings.

Reply:
If you want to silence the warnings, go ahead (even though by default in the Makefile they aren't enabled, so they probably only appear in your setup).

Reply:
argc/argv is not implemented and will probably not be implemented on ARM7, so providing it in the `main()` function signature feels a little deceptive/misleading.

Reply:
Well, fair enough, we can leave the arguments out. We can change it back if they are used for something at some point.